setup.py

```diff
@@ -52,7 +52,7 @@ def package_data(pkg, roots):
         'requests>=2.9.1,<3.0.0',
         'babelfish>=0.5.5,<0.6.0',
         'XBlock>=1.4.0,<2',
-        'xblock-utils>=2.1,<4'
+        'xblock-utils==4.0.0'
```

Comment:
This XBlock is deprecated. But the functionality is imported from the new Xblock and we can use it.
https://github1s.com/openedx-unsupported/xblock-utils/blob/v4.0.0/xblockutils/studio_editable.py

Reply:
Why didn't you remove this dependency and replace its usage with the XBlock package (where the code was migrated)?

`'XBlock>=1.4.0,<2',` also conflicts with edx-platform's requirements where it is pinned on `xblock[django]==4.0.1` and this version [contains the migrated code](https://github.com/openedx/XBlock/tree/v4.0.1/xblock/utils) 

Please update your PR:
- remove deprecated xblock-utils
- replace its usage with the corresponding XBlock code 
- update the XBlock requirement corresponding to the edx-platform version
- re-check if there are any other outdated packages to update compared to edx-platform requirements

Reply:
done